### ISSUE TITLE
Accessibility Issue: IMG does not have an alt text /buy-vs-build-start.html

### ISSUE THREAD

**Issue by Moro-Code** [sara-sabr/ITStrategy#738]
**Description**
Every image must have a descriptive alt text 

**Device Information**  
*Device*:  
*Operating System (incl. version)*:  
*Browser (incl. version)*:  
*Additional Details*:

**Element(s) Information**  
*Page*:  https://sara-sabr.github.io/ITStrategy/buy-vs-build-start.html
*CSS Selector(s) (if possible and applicable)*:  body > main > p:nth-child(4) > a > img
*Additional Identifier(s) (if possible and applicable)*:  
*Additional Details*:  

**Screenshot(s) (if possible)**

<img width="538" alt="Screen Shot 2020-01-31 at 10 32 27 AM" src="https://user-images.githubusercontent.com/26819992/73552024-75d33780-4415-11ea-9375-b5238dea8fa0.png">

**Additional information or thoughts**

As a result of #699 

https://dequeuniversity.com/rules/axe/2.6/image-alt
